docs/solutions/sql_server_always_on_availability_groups/modules/network/subnet/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "oci_core_subnet" "subnet" {
+  count      = "${var.ad_count}"
+  cidr_block = "${var.cidr_block["${count.index}"]}"
+
+  // Some terraform trickery required here.
+  // Due to the fact that we need to specify placement for the Witness server, we're using additional variable ${ad_placement}
```

Comment:
I think you mean ad_deployment?

Reply:
yes. the Witness instance can exist in any AD